src/App.test.js

```diff
@@ -1,8 +1,8 @@
 import { render, screen } from '@testing-library/react';
 import App from './App';
 
-test('renders learn react link', () => {
+test('renders brand link', () => {
```

Comment:
Por lo general todos los tests se meten en una carpeta al nivel de src, por ejemplo src/tests/App.test.js o para otros archivos podría ser src/test/componentes/Home.test.js y así es mas fácil identificar los tests.

Reply:
Good. So, in this case I can focus to specific tests for each component ?
In this case only test the Header component for asserting the presence of the brand.